tests/src/Context/DrushContext.php

```diff
@@ -76,6 +76,7 @@ public static function prepareDatabaseForFeatureTest(BeforeFeatureScope $scope)
     else {
       self::dropDataBase();
       self::importDataBase();
+      self::resetStatic();
```

Comment:
Static variables were being kept from feature to feature, resulting in many unexpected behaviors.
This is a great improvement to our behat tests. It doesn't come directly from the update of drupal-extension, but we needed this to make tests work again. This had side impacts in other places.